src/components/CollaboratorModal/components/AcknowledgementSubmodal.tsx

```diff
@@ -93,23 +78,17 @@ const AcknowledgementSubmodalContent = () => {
         </Text>
       </Checkbox>
       <Stack spacing={4} direction="row" justify="right">
-        <Button
-          variant="clear"
-          color="secondary"
-          onClick={() => {
-            setModalState(CollaboratorModalState.Default)
-            setIsAcknowledged(false)
-            closeModal()
-          }}
-        >
+        <Button variant="clear" color="secondary" onClick={onClose}>
```

Comment:
I think this is going to be quite nitty and annoying: The `isAcknowledged` should reset to false on cancel. Otherwise, it is possible for the user to "bypass" the acknowledgement. Steps to reproduce:

1. Add a collaborator that requires acknowledgement
2. The acknowledgement submodal appears, click on the "I agree to Isomer terms of use" checkbox
3. Click on "Cancel" or the close button (this indicates that the user has decided to disagree with the terms of use)
4. Open the collaborators modal again, the previous collaborator is prefilled in the input box (separate issue which I will address in another comment)
5. Click on "Add collaborator"
6. Notice that the collaborator is added without the acknowledgement submodal appearing. It should appear to prompt the user to agree to the terms of use.

I think this is being super nitty already so if no time can just skip this. No loss in functionality anyway if we do not fix this.

Reply:
clarified offline - happens when 
1. fill in collaborator which requires ACK
2. go to ack modal and tick
3. click on the **closebutton**
4. reopen modal 
5. fill in same collaborator and click add
6. toast pops up

Reply:
ok turns out `reset` was from the `useMutation` hook not `react-hook-forms` so the state wasn't reset after modal closes .-. 

fixed